buildconfigs/key_xor_test_app.toml

```diff
@@ -10,3 +10,4 @@ command = [
   "--release",
 ]
 artifact_path = "./enclave_apps/target/x86_64-unknown-none/release/key_xor_test_app"
+package_name = "key_xor_test_app"
```

Comment:
Is the idea that the package would contain several binaries, in the case of a wildcard in the artifact path? Is there a canonical documentation / reference for the meaning of those fields?

Reply:
And what's the difference (if any) between the file name of the build config and the package name? Or is the file name not used at all?

Reply:
We should not use wildcards here (since it is "path", for one), but I need to investigate the consequences.

The name of the buildconfig file is not used at all.

Reply:
> And what's the difference (if any) between the file name of the build config and the package name?

Could you answer this too? How would we (or other users of these files) pick the binary name vs the package name vs the filename of the build config file?